src/index.tsx

```diff
@@ -103,7 +103,7 @@ if (!window || !window._env_) {
         SENTRY_TRACES_SAMPLE_RATE: 0.2,
         HOTJAR_ENABLED: false,
         GA_ENABLED: false,
-        APPLICATION_METRICS_ENABLED: false,
+        APPLICATION_METRICS_ENABLED: true,
```

Comment:
Is this expected? Do we want to enable `APPLICATION_METRICS_ENABLED` by default in pre-prod?

Reply:
It will enable it for local only